src/Login/index.jsx

```diff
@@ -0,0 +1,148 @@
+import React, { Component, PropTypes } from 'react';
+import { themr } from 'react-css-themr';
+import { connect } from 'react-redux';
+import {
+    addLocaleData,
+    IntlProvider,
+    FormattedMessage,
+    defineMessages
+} from 'react-intl';
+import enLocaleData from 'react-intl/locale-data/en';
+import zhLocaleData from 'react-intl/locale-data/zh';
+
+import { setUsername, setPassword, toggleRemember, attemptLogin } from './actions';
+import Input from '../Form/Input';
+import style from './style.scss';
+import constants from '../constants';
+import zhMessage from './l10n/zh_CN';
+import enMessage from './l10n/en_US';
+
+addLocaleData(enLocaleData);
+addLocaleData(zhLocaleData);
```

Comment:
Are we bundling all of the locale data inside webpack? Won't this make our exports huge?